VMware.CloudFoundation.CertificateManagement.psm1

```diff
@@ -53,6 +53,33 @@ Function Get-Password {
     return $password
 }
 
+function Restart-VcenterService {
+    <#
+    .DESCRIPTION
+        The Restart-VcenterService restart the vcenter service taken from parameter value and returns an object arrary with the service name, the service status (started or stopped) and the result from restart operation.  
+    #>
```

Comment:
```suggestion
     
     .PARAMETER serviceName
     The name of the service on vCenter Server.
    
    #>
```

Reply:
addressed in commit eb037523e0d6150f6f688c31fbde63553b588cdf